### PR TITLE
Update instructions for `register-ci` and `regenerate`, fix typos, change 'regenerate' to 'rerender'

### PR DESCRIPTION
**Pull request by shadowwalkersb**
The commands as they are currently did not work for me. But, they worked after fixing them as in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ This requires a github token. You can try it out with a github user account
 instead of an organization by replacing the organization argument with
 `--user github_user_name`.
 3. **Register the feedstock with CI services:**
-`conda smithy register-ci --organization conda-forge ./foo-feedstock`.
+`conda smithy register-ci --organization conda-forge --feedstock_directory ./foo-feedstock`.
 This requires tokens for the CI services. You can give the name of a user instead
 of organization with `--user github_user_name`.
-4. Regenerate the feedstock with ``conda smithy regenerate ./foo-feedstock``
+4. Regenerate the feedstock with ``conda smithy regenerate --feedstock_directory ./foo-feedstock``
```
Comment (jakirkham) [inline README.md]: Mostly we use Re-render these days (though it is an alias). Could we please update this to reference `rerender` as well?

**Comment by jakirkham** [comment]
Added a minor suggestion. Otherwise 👍.

File: README.md (hunk 1 above)
Comment (shadowwalkersb) [inline README.md]: Do you mean replace `regenerate` with `rerender`?

Comment (pelson) [inline README.md]: Yes please.

**Comment by pelson** [comment]
Thanks for doing this. 👍

**Comment by shadowwalkersb** [comment]
I also fixed a tiny typo in the travis template. Let me know if it is wrong or if it needs to go in a separate PR. I'll revert it here.

**Comment by jakirkham** [comment]
Nope that is correct. Thanks for fixing it. Also there is a line in the AppVeyor [template]( https://github.com/conda-forge/conda-smithy/blob/611d4ecbebb28012197a51f70a89b0463a2dabaf/conda_smithy/templates/appveyor.yml.tmpl#L2 ) that needs fixing too.

**Comment by shadowwalkersb** [comment]
Anything else, while we are at it?

**Comment by jakirkham** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -46,10 +46,10 @@ This requires a github token. You can try it out with a github user account
 instead of an organization by replacing the organization argument with
 `--user github_user_name`.
 3. **Register the feedstock with CI services:**
-`conda smithy register-ci --organization conda-forge ./foo-feedstock`.
+`conda smithy register-ci --organization conda-forge --feedstock_directory ./foo-feedstock`.
 This requires tokens for the CI services. You can give the name of a user instead
 of organization with `--user github_user_name`.
-4. Regenerate the feedstock with ``conda smithy regenerate ./foo-feedstock``
+4. Regenerate the feedstock with ``conda smithy rerender --feedstock_directory ./foo-feedstock``
```
Comment (jakirkham) [inline README.md]: Might as well change `Regenerate` to `Re-render` if we are changing the CLI call down the line.

File: conda_smithy/templates/README.md.tmpl [hunk 3]
```diff
@@ -52,7 +52,7 @@ packages to the [conda-forge](https://anaconda.org/conda-forge)
 To manage the continuous integration and simplify feedstock maintenance
 [conda-smithy](http://github.com/conda-forge/conda-smithy) has been developed.
 Using the ``conda-forge.yml`` within this repository, it is possible to regenerate all of
```
Comment (shadowwalkersb) [inline conda_smithy/templates/README.md.tmpl]: How about here?

Comment (jakirkham) [inline conda_smithy/templates/README.md.tmpl]: 👍 Go for it.

**Comment by jakirkham** [comment]
Let's fix that one last thing and handle the merge conflicts.

**Comment by shadowwalkersb** [comment]
Merge conflicts handled, did one last change from `regenerate` to `rerender` and updated the title of PR.

**Comment by shadowwalkersb** [comment]
ping

**Comment by jakirkham** [comment]
Awesome stuff, @shadowwalkersb. Have been wanting to fix this forever. Thanks for taking it on.

**Pull request merged**